app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -66,7 +66,7 @@ class PatcherViewModel(
     private val pm: PM by inject()
     private val workerRepository: WorkerRepository by inject()
     private val installedAppRepository: InstalledAppRepository by inject()
-    private val rootInstaller: RootInstaller by inject()
+    val rootInstaller: RootInstaller by inject()
```

Comment:
Make `rootInstaller` private and provide `hasRootAccess` through the viewmodel instead

Reply:
Done with https://github.com/ReVanced/revanced-manager/commit/b6dc75ae9984322b6ea2b3ad66d00c48c48751f0